exercises/concept/interest-is-interesting/.docs/instructions.md

```diff
@@ -0,0 +1,58 @@
+# Instructions
+
+In this exercise you'll be working with savings accounts. Each year, the balance of your savings account is updated based on its interest rate. The interest rate your bank gives you depends on the amount of money in your account (its balance):
+
+- 3.213% for a negative balance (balance gets more negative).
+- 0.5% for a positive balance less than `1000` dollars.
+- 1.621% for a positive balance greater than or equal to `1000` dollars and less than `5000` dollars.
+- 2.475% for a positive balance greater than or equal to `5000` dollars.
+
+You have four tasks, each of which will deal your balance and its interest rate.
+
+## 1. Calculate the interest rate
+
+Implement the `interest_rate()` method to calculate the interest rate based on the specified balance:
+
+```rust
+interest_is_interesting::interest_rate(200.75)
```

Comment:
Could we lose the `interest_is_interesting::` prefix? I don't see it being used in `lasagna` and the student can just implement the stub so they don't need to know about the `interest_is_interesting::` part I think